app/src/main/java/mollie/tictactoe/AndroidPlayerType.java

```diff
@@ -0,0 +1,8 @@
+package mollie.tictactoe;
+
+import ttt.PlayerType;
+
+public class AndroidPlayerType extends PlayerType {
```

Comment:
Seeing the suffix -Type on a class has a slight smell to it. Classes define types. The variable HUMAN is of type string. But you don't see that thing being called StringType. or IntegerType.